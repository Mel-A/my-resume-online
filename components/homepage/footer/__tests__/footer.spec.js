import { shallow } from 'enzyme'
import Footer from '..'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}))

test('footer renders correctly', () => {
  const footer = shallow(<Footer />)
  expect(footer).toMatchSnapshot()
})