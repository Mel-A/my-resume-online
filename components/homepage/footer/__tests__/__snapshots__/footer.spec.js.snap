// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer footer renders correctly 1`] = `ShallowWrapper {}`;

exports[`footer footer renders theme correctly 1`] = `ReactWrapper {}`;
